modules/ROOT/pages/con_pipeline-cluster-resource.adoc

```diff
@@ -0,0 +1,48 @@
+[]
+= Cluster Resource
+
+
+A Cluster Resource represents a Kubernetes cluster other than the current cluster Pipelines is running on. A common use case for this resource is to deploy your application or function on different clusters.
```

Comment:
Also, should we talk about use cases here? I am not sure. I would rather say, "With this resource,  you can deploy your application or function on different clusters." What do you think?